apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -899,14 +917,15 @@ impl Webserver {
                                 route_table.insert(
                                     after.path.clone(),
                                     RouteMeta {
-                                        format: after.format.clone(),
+                                        // When doing Ingress API the format is always known
+                                        format: after.format.unwrap().clone(),
```

Comment:
If so we should move format to be part of `APIType::INGRESS`

Reply:
Ha yeah good point